core/commands/dag/import.go

```diff
@@ -176,6 +181,7 @@ func importWorker(req *cmds.Request, re cmds.ResponseEmitter, api iface.CoreAPI,
 				if err := batch.Add(req.Context, nd); err != nil {
 					return err
 				}
+				blockCount++
```

Comment:
@rvagg now seeing this and the output, I am wishing for a `payloadBytesCount++` as well 🤔